File: Python/Pc.py
class Pc:

    def __init__(self, p, d, r):
        self.objProcee = p
        self.objDisk = d
        self.objRam = r
        self.totalPrice = self.objProcee.getPrice() + d.getPrice() + r.getPrice()

    def getTotalPrice(self):
        return self.totalPrice

    def printIni(self):
        print("----------------------")
        print("Detail PC")
        self.objProcee.printDetProcee()
        self.objDisk.printDetDisk()
        self.objRam.printDetRam()
        print("----------------------")
        print("Total Price    : ", self.totalPrice)

File: Java/Pc.java
class Pc {

    private int totalPrice;
    Processor pObj;
    Disk dObj;
    Ram rObj;

    Pc() { }
    Pc(Processor p, Disk d, Ram r){
        this.pObj = p;
        this.dObj = d;
        this.rObj = r;
    }


    public int getTotalPrice() {
        totalPrice = this.pObj.getPrice() + this.dObj.getPrice() + this.rObj.getPrice();
        return totalPrice;
    }
    // ganti isi methodnya
    public void setTotalPrice() {
        this.totalPrice = this.pObj.getPrice() + this.dObj.getPrice() + this.rObj.getPrice();
    }

    void printDetPc(){
        System.out.println("Detail PC");
        this.pObj.printDetProcee();
        this.dObj.printDetDisk();
        this.rObj.printDetRam();
        System.out.println("----------------------");
        System.out.println("Total Price    : " + getTotalPrice());
    }

    
}
